openmc/cell.py

```diff
@@ -79,13 +79,12 @@ class Cell(object):
     translation : Iterable of float
         If the cell is filled with a universe, this array specifies a vector
         that is used to translate (shift) the universe.
-    offsets : ndarray
-        Array of offsets used for distributed cell searches
-    distribcell_index : int
-        Index of this cell in distribcell arrays
-    distribcell_paths : list of str
-        The paths traversed through the CSG tree to reach each distribcell
-        instance
+    paths : list of str
+        The paths traversed through the CSG tree to reach each cell
+        instance. This property is initialized by calling the
+        :meth:`Geometry.determine_paths` method.
```

Comment:
I'm still not clear about this - it appears that you've made this method private, and that it is called by default. If this is what you intend, then the docstring here and in "material.py" should be updated accordingly; otherwise, it should be made public and callable by the user rather than by default.